benchmarks/hash_table/static_multimap/count_bench.cu

```diff
@@ -27,21 +27,22 @@
  * - Total number of insertions: 100'000'000
  * - CG size: 8
  */
-template <typename Key, typename Value, dist_type Dist, nvbench::int32_t Multiplicity>
+template <typename Key, typename Value>
```

Comment:
Thanks for doing the multimap cleanups. I assume this will decrease the build time also?

Reply:
Indeed, the compile time went down significantly. There is no real reason to have this as a compile-time parameter since there is no `switch constexpr`. It also simplifies the benchmark interface.